esbm/esbm.html

```diff
@@ -23368,11 +23368,11 @@
 <hr/>
-<p class="ms-0">◎<strong style="color: darkblue">*</strong><strong class="hw" id="lieshkac" style="color: darkblue">Лешкаць</strong>, <span style="color: brown">лэшкаты</span> ’пакладаць, кастрыраваць’ (<v-abbr>кобр.</v-abbr>, На] лекс.). Інфікс <span style="color: brown">‑к‑</span> перадае аднаразовасць дзеяння (як, напрі клад, <v-abbr>драг.</v-abbr> <span style="color: brown">пэцатэ</span> — <span style="color: brown">пэцкатэ</span> ’пэцкаць’). Лексема ўзыходзіі да <v-abbr>рус.</v-abbr> <span style="color: brown">лишать</span> ’пазбаўляць чаго-небудзь’. У суседніх дра гаворках — <span style="color: brown">лэчатэ</span> ’пакладаць’ (І. Лучыц-Федарэц ).</p>
+<p class="ms-0">◎<strong style="color: darkblue">*</strong><strong class="hw" id="lieshkac" style="color: darkblue">Лешкаць</strong>, <span style="color: brown">лэ́шкаты</span> ’пакладаць, кастрыраваць’ (<v-abbr>кобр.</v-abbr>, <v-abbr class="source">Нар. лекс.</v-abbr>). Інфікс <span style="color: brown">‑к‑</span> перадае аднаразовасць дзеяння (як, напрыклад, <v-abbr>драг.</v-abbr> <span style="color: brown">пэцатэ</span> — <span style="color: brown">пэцкатэ</span> ’пэцкаць’). Лексема ўзыходзіць да <v-abbr>рус.</v-abbr> <span style="color: brown">лишать</span> ’пазбаўляць чаго-небудзь’. У суседніх <v-abbr>драг.</v-abbr> гаворках — <span style="color: brown">лэ́чатэ</span> ’пакладаць’ (І. Лучыц-Федарэц).</p>
 <hr/>
-<p class="ms-0">◎<strong class="hw" id="lieshuki" style="color: darkblue">Лешукі́</strong> ’лесуны, лясныя нячысцікі’ (<v-abbr class="source">Нік., Няч.</v-abbr>). Рэгіянальнае <v-abbr>бел.</v-abbr> утварэнне з суф. <span style="color: brown">‑ук</span> ад <a href="#liot">лёт</a>- (параўн. <v-abbr>гродз.</v-abbr> <span style="color: brown">леты</span> (<v-abbr class="source">Сл. паўн.-зах.</v-abbr>), <v-abbr>рус.</v-abbr> <span style="color: brown">леший des‑jь</span>).</p>
+<p class="ms-0">◎<strong class="hw" id="lieshuki" style="color: darkblue">Лешукі́</strong> ’лесуны, лясныя нячысцікі’ (<v-abbr class="source">Нік. Няч.</v-abbr>). Рэгіянальнае <v-abbr>бел.</v-abbr> утварэнне з суф. <span style="color: brown">‑ук</span> ад <a href="#liot">лёт</a>- (параўн. <v-abbr>гродз.</v-abbr> <span style="color: brown">ле́шы</span> (<v-abbr class="source">Сл. паўн.-зах.</v-abbr>), <v-abbr>рус.</v-abbr> <span style="color: brown">леший &lt; lěs‑jь</span>).</p>
```

Comment:
```suggestion
<p class="ms-0">◎<strong class="hw" id="lieshuki" style="color: darkblue">Лешукі́</strong> ’лесуны, лясныя нячысцікі’ (<v-abbr class="source">Нік. Няч.</v-abbr>). Рэгіянальнае <v-abbr>бел.</v-abbr> утварэнне з суф. <span style="color: brown">‑ук</span> ад <span style="color: brown">леш-</span> (параўн. <v-abbr>гродз.</v-abbr> <span style="color: brown">ле́шы</span> (<v-abbr class="source">Сл. паўн.-зах.</v-abbr>), <v-abbr>рус.</v-abbr> <span style="color: brown">леший &lt; lěs‑jь</span>).</p>
```

<img width="1107" alt="image" src="https://github.com/user-attachments/assets/07126722-9a59-40a0-b562-dd85a55f25b1">

https://knihi.com/none/Etymalahicny_slounik_bielaruskaj_movy_zip.html#ESBM_05a.djvu_293